bibtex-actions.el

```diff
@@ -489,7 +490,7 @@ TEMPLATE."
   "Return citation keys at point as a list for `embark'."
   (when-let ((key (or (bibtex-actions-get-key-org-cite)
                       (bibtex-completion-key-at-point))))
-    (cons 'citation-key (if (listp key) key (list key)))))
+    (cons 'citation-key (if (listp key) (string-join key " & ") key))))
```

Comment:
Actually, one problem I see @ilupin:

When doing `embark-act` on a single key and choosing the default action (`RET`), it returns a type error. Same with doing `embark-dwim`.

But the command it runs (by default `bibtex-actions-open`) does not return that error if I choose it directly.

If I change the above to this, I still get an error.

``` elisp
    (cons 'citation-key (if (listp key) (string-join key " & ") (list key)))))
```

Reply:
Fixed in https://github.com/bdarcus/bibtex-actions/pull/171/commits/8296dbaa7f0ab2a315705c8fabf431c4ad704fa7

On `bibtex-actions-run-default-action` just needed to split if a string.